package.json

```diff
@@ -119,9 +119,9 @@
     "start-cordova": "REACT_APP_ENV=cordova REACT_APP_WHEELMAP_API_BASE_URL=https://wheelmap.org PUBLIC_URL=\"./\"",
     "analyze": "source-map-explorer build/static/js/main.*",
     "test": "node scripts/test.js --env=jsdom",
-    "build": "extract=1 BABEL_DISABLE_CACHE=1 node scripts/build.js",
-    "build-cordova": "REACT_APP_ENV=development REACT_APP_WHEELMAP_API_BASE_URL=https://wheelmap.org PUBLIC_URL=\"./\" node scripts/build.js; rm -rf www; cp -r build www",
-    "build-staging": "REACT_APP_ENV=staging node scripts/build.js",
+    "build": "extract=1 BABEL_DISABLE_CACHE=1 REACT_APP_TAGS_TO_APPEND_TO_INDEX_HTML_HEAD=\"\" node scripts/build.js",
+    "build-cordova": "REACT_APP_TAGS_TO_APPEND_TO_INDEX_HTML_HEAD=\"<script src='cordova.js'></script>\" REACT_APP_ENV=development NODE_ENV=development CORDOVA=1 REACT_APP_WHEELMAP_API_BASE_URL=https://wheelmap.org PUBLIC_URL=\"./\" node scripts/build.js",
```

Comment:
maybe build-cordova-dev?

Reply:
I meant to use this for production builds too – we do not have a 'real' development environment for cordova yet. Can we live with unminified app builds for now?

Reply:
Sure